components/table/demo/table-test.js

```diff
@@ -98,7 +145,7 @@ class TestTable extends RtlMixin(LitElement) {
 	_renderSortButton(fruit) {
 		const noSort = this._sortField !== fruit.toLowerCase();
 		return html`
-			<th>
+			<th scope="col">
```

Comment:
These `scope` changes aren't strictly necessary, just good hygiene (borrowed from the draft PR that was adding support for sticky + scroll-wrapper).